packages/compiler-cli/src/ngtsc/annotations/component/src/handler.ts

```diff
@@ -481,7 +482,9 @@ export class ComponentDecoratorHandler implements
                 node, this.reflector, this.isCore, this.annotateForClosureCompiler,
                 dec => transformDecoratorResources(dec, component, styles, template)) :
             null,
-        classDebugInfo: extractClassDebugInfo(node, this.reflector, this.rootDirs),
+        classDebugInfo: extractClassDebugInfo(
+            node, this.reflector, this.rootDirs,
+            !metadata.isStandalone && this.forbidOrphanRendering),
```

Comment:
for more clarity / or we use a keyed options object (same in the test)

```suggestion
           /* forbidOrphanRenderering */ !metadata.isStandalone && this.forbidOrphanRendering),
```

Reply:
Do we just want to pass the option and let runtime deal with the standalone case? (that would seem more consisten to me)